src/main/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategy.java

```diff
@@ -168,4 +372,19 @@ void setLocalRegion(String localRegion)
     {
         this.localRegion = localRegion;
     }
+
+    public JSONObject getStats()
+    {
+        JSONObject json = new JSONObject();
+
+        json.put("total_a1", totalA1);
```

Comment:
I think these should be more descriptive as well, otherwise we'll have to look them up every time.

Reply:
yeah agreed, it doesn't mean anything like this. Any good name suggestions?

Reply:
I think the names you used for the methods are good (tweaked a bit...without the leading `find`, for example)